src/cls/tabular/cls_tabular.cc

```diff
@@ -78,7 +111,13 @@ int build_sky_index(cls_method_context_t hctx, bufferlist *in, bufferlist *out)
     // a ceph property encoding the len of each bl in front of the bl,
     // seems to be an int32 currently.
     const int ceph_bl_encoding_len = sizeof(int32_t);
-    int fb_seq_num = 0;  // TODO: get this from a stable counter.
+
+    int fb_seq_num;
```

Comment:
Please initialize to the default value
Tables::FB_SEQ_NUM_MIN
since you are not setting this via function return value now.